src/AdminPage.js

```diff
@@ -194,21 +195,16 @@ export const AdminPage = () => {
   const data = useLoaderData();
 
   return (
-    <>
-      <React.Suspense
-        fallback={
-          <p>
-            <Spinner></Spinner>Loading...
-          </p>
-        }
-      >
-        <Await
-          resolve={data.singleEvent}
-          errorElement={<p>An error occurred</p>}
-        >
-          <AdminChild />
-        </Await>
-      </React.Suspense>
-    </>
+    <React.Suspense
+      fallback={
+        <p>
```

Comment:
I'm getting a console error which seems like it's regarding this `<p>` tag:
`"<div> cannot appear as a descendant of <p>."`

Reply:
I bet the Loading component has a div in it--makes sense, taking this out.